module-4/backend/week_one.md

```diff
@@ -5,15 +5,34 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+For me, I try to take the intermission work as initial exposure to whatever concepts we'll be learning during the mod. I try to digest the material as muscle memory and syntax exposure.
 2. What are some tools to help debug JavaScript code?
+DevTools, debugger, pryjs, console.log
 3. What are some tools you need in order to unit test your JavaScript?
+mocha
 4. What is the syntax for invoking a function?
+function x( , {
+
+})
+OR
+const x = () => {
+
+}
 5. What's `this` in JavaScript?
+this is whatever it is being called by. meta...
 6. What is Webpack and why is it useful?
+webpack is the compiler for javascript, kind of like the asset pipeline.
 7. When/why do you want to use event delegation?
+it allows you to use a single event listener, but interact with it on multiple layers.
```

Comment:
Yes, you often want to use event delegation for HTML Elements that don't currently exist on the page but may in the future.